src/sdg/producten/admin/product.py

```diff
@@ -149,6 +155,12 @@ def is_referentie(self, obj):
 
     is_referentie.boolean = True
 
+    def _referentie_product(self, obj):
```

Comment:
We usually do `get_`
```suggestion
    def get_referentie_product(self, obj):
```